tests/cypress/tests-e2e/drawing.cy.js

```diff
@@ -284,6 +302,17 @@ describe('Drawing module tests', () => {
                     new RegExp(`<LabelStyle><color>${KML_STYLE_BLACK}</color></LabelStyle>`),
                 ])
             })
+
+            cy.get('[data-cy="drawing-style-delete-button"]').click({ force: true })
+            cy.clickDrawingTool(EditableFeatureTypes.ANNOTATION)
+            cy.get('[data-cy="ol-map"]:visible').click()
+            readCoordinateClipboard('feature-style-edit-coordinate-copy')
```

Comment:
Here instead of deleted the previous feature and re-add a new one, I would leave the previous one, add a new at a different coordinate and test the copy of both feature making sure that the clipboard contains each time the correct coordinate. So we make sure the a the clipboard gets updated on a subsequent click (used to be a bug on the location popup)